config_examples/config.json

```diff
@@ -1,6 +1,7 @@
 // This file contains full features of Ddosify as a reference. Don't use it directly.
 {
     "request_count": 20,
```

Comment:
What about removing "request_count" or commenting as "This field will be deprecated, please use iteration_count instead"?